include/eve/conditional.hpp

```diff
@@ -126,11 +130,23 @@ namespace eve
     static constexpr bool is_inverted     = false;
     static constexpr bool is_complete     = true;
 
+    template<typename V> EVE_FORCEINLINE auto else_(V v) const  {  return or_(*this,v);  }
+
     template<typename T> EVE_FORCEINLINE auto mask(eve::as_<T> const&) const
     {
       return eve::as_logical_t<T>(false);
     }
 
+    template<typename T> EVE_FORCEINLINE std::ptrdiff_t offset(eve::as_<T> const&) const
+    {
+      return 0;
+    }
+
+    template<typename T> EVE_FORCEINLINE auto count(eve::as_<T> const&) const
+    {
+      return 0ULL;
+    }
```

Comment:
Why is this unsigned but offset is signed?

Reply:
this is a size and is usually computed from sizeof in other case, hence the 0ULL.
Offset is maybe a good candidate to become a size_t too.

Reply:
Not sure I agree, but it doesn't really matter.